src/groups/universe.rs

```diff
@@ -205,4 +224,14 @@ impl<'a> UniverseGroup<'a> {
         Structure,
         (structure_id: u64) => "{structure_id}"
     );
+
+    api_post!(
+        /// Get IDs from a list of names
+        get_ids,
+        "post_universe_ids",
+        RequestType::Public,
+        Ids,
+        ,
+        names: &Vec<String>,
```

Comment:
Let's try making this a more permissive type:

```suggestion
names: &[&str],
```

Does that work for your code?